pkg/codegen/internal/test/testdata/simple-enum-schema/dotnet/Enums.cs

```diff
@@ -0,0 +1,38 @@
+// *** WARNING: this file was generated by test. ***
+// *** Do not edit by hand unless you're certain you know what you are doing! ***
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Threading.Tasks;
+using Pulumi.Serialization;
+
+namespace Pulumi.PlantProvider.Enums
+{
+    /// <summary>
+    /// plant container colors
+    /// </summary> 
+    public enum ContainerColor
+    {
+        [EnumMember(Value = "red")]
+        Red,
+        [EnumMember(Value = "blue")]
+        Blue,
+        [EnumMember(Value = "yellow")]
+        Yellow
+    }
+
+    /// <summary>
+    /// plant container sizes
+    /// </summary>    
+    public enum ContainerSize
+    {
+        [EnumMember(Value = 4)]
+        FourInch,
```

Comment:
C# enums are integers by default, so we should assign int values to options if our underlying type is int.
```suggestion
        FourInch = 4,
```

This way users can cast between enum and int without extra ceremony.